### PR TITLE
add mc info as the key of sensor instance

### PR DESCRIPTION
**Pull request by PaynePei**

### CODE REVIEW

**Comment by emcinfrasim** [comment]
```
******* Functional Test ******* 
TestName: test_sensor_accessibility 
ErrorMessage: exceptions.AssertionError 
ErrorINfo: Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/case.py", line 329, in run
    testMethod()
  File "/home/infrasim/workspace/Infrasim_Compute_pull_request_trigger/infrasim-compute/test/functional/test_ipmi_console.py", line 97, in test_sensor_accessibility
    assert 'Fan_SYS0' in str_output
AssertionError
 
```

**Comment by PaynePei** [comment]
@MarkMa001 @helloarys please have a review

**Comment by fub2** [comment]
Functional test is failing. Why "All checks have passed" shows up?

**Comment by fub2** [comment]
Merge the code in order for building OVA out before build server shutdown. 
Please review the code as appropriate. @turtle-fly

File: infrasim/ipmicons/command.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def __init__(self):
     def add_msg(self, msg):
         logger.info(msg)
 
-    def get_sensor_instance(self, str_num):
+    def get_sensor_instance(self, str_num, mc=int("0x20", 16)):
```
Comment (turtle-fly) [inline infrasim/ipmicons/command.py]: mc = 0x20
is a shorter expression~

**Comment by turtle-fly** [comment]
ok with this change @PaynePei 
:+1:

File: infrasim/ipmicons/command.py (hunk 1 above)
Comment (emcinfrasim) [inline infrasim/ipmicons/command.py]: good suggestion

**Pull request merged**